.vimrc

```diff
@@ -0,0 +1,10 @@
+filetype plugin indent on
```

Comment:
this file should me removed from this PR. A discussion about it is happening here https://github.com/superdesk/superdesk-client-core/pull/966